Libs/Common/Shapeworks.h

```diff
@@ -133,7 +133,7 @@ P operator-(const P &p, const P &q)
 template<typename P, typename = std::enable_if_t<std::is_same<Image, P>::value ||
                                                  std::is_same<Coord, P>::value ||
                                                  std::is_same<Dims, P>::value ||
-                                                 std::is_same<Vector, P>::value ||
+                                                 std::is_same<Vector, P>::value || // use operator*(v0, v1); (or call dotProduct)
```

Comment:
This isn't super clear, though I know what is meant: `v0 * v1` won't call this function, so if you want to use it, you have to call `operator*(v0, v1)` or just use the `dotProduct` function. Clarification or suggestions are welcome.

Reply:
This is definitely not a reason to postpone a merge. We only added a comment to the `enable_if`.